contracts/settling_game/L02_Resources.cairo

```diff
@@ -118,44 +126,53 @@ func claim_resources{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_che
     # check days greater than zero
     assert_not_zero(days)
 
+    # get wonder tax percentage 
+    let ( wonder_tax ) = I04A_Calculator.calculateWonderTax(contract_address=calculator_address)
+    let wonder_tax_rel_perc = (80 * wonder_tax) / 100
+
+    # set minting percentages
+    let treasury_mint_perc = 20 + wonder_tax_rel_perc
+    let user_mint_rel_perc = 80 - wonder_tax_rel_perc
+
     assert resource_ids[0] = realms_data.resource_1
-    assert user_mint[0] = ((r_1 * days) * 80) / 100
-    assert treasury_mint[0] = ((r_1 * days) * 20) / 100
+    assert user_mint[0] = ((r_1 * days) * user_mint_rel_perc) / 100
```

Comment:
DRY: precalculate `(days * user_mint_rel_perc) / 100` and `(days * treasury_mint_perc) / 100`.